vita3k/ctrl/src/ctrl.cpp

```diff
@@ -280,11 +281,11 @@ static void retrieve_ctrl_data(EmuEnvState &emuenv, int port, bool is_v2, bool n
             rx = float_to_byte(axes[2]);
             ry = float_to_byte(axes[3]);
         }
-    };
-
-    if (emuenv.common_dialog.status == SCE_COMMON_DIALOG_STATUS_RUNNING) {
         if (negative)
             buttons ^= ~0;
+    };
+
+    if ((emuenv.common_dialog.status == SCE_COMMON_DIALOG_STATUS_RUNNING) || emuenv.drop_inputs) {
```

Comment:
Why the case (emuenv.common_dialog.status == SCE_COMMON_DIALOG_STATUS_RUNNING) is not taken into account in drop_inputs?

Reply:
? Is just normal formating code ?
If have one == and adding other bool with OR i need separe it

Reply:
drop_inputs only purpose is to be set to true when you drop the input. But when there is a common dialog running you drop the input but don't set it to true.

Reply:
Ah i understund sorry
So yes is normal, common dialog is diferent of other dialog relate drop input
Need 2 separate for good reason
No forget common dialog using (for save dialog only for now)  input for controle it with input
If you look interface, i only drop input for settings dialog, not for common dialog, is normal

Reply:
Don't name a variable drop_input if it being set to false doesn't mean we don't drop input.

Reply:
Eu what ? Is not my name and what ? 🤔
Ifcis false is normal it no drop 🤔